invoice/models.py

```diff
@@ -140,7 +140,7 @@ def items(self):
     @property
     def table_export(self):
         """Get the line items as list and export all of them as table with a header row."""
-        header = ['Name', 'Description', 'Quantity', 'Price', 'Tax', 'Net Total', 'Total']
+        header = ['Name', 'Description', 'Quantity', 'Unit', 'Price', 'Tax', 'Net Total', 'Total']
```

Comment:
Can we combine unto with quantity, like the money amounts?
Would save some space in the table.

Reply:
oh yeah, great idea